### PR TITLE
EFK for 19.10

### PR DESCRIPTION
**Pull request by fatpat**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-openio_filebeat_namespace: OPENIO
-openio_filebeat_gridinit_dir: "/etc/gridinit.d/{{ openio_filebeat_namespace }}"
-openio_filebeat_gridinit_file_prefix: ""
-openio_filebeat_pid_directory:
-  "/run/filebeat/{{ openio_filebeat_namespace }}/filebeat-{{ openio_filebeat_serviceid }}"
+openio_filebeat_version: 7.5.0
 
 openio_filebeat_serviceid: "0"
-openio_filebeat_volume: "/var/lib/oio/sds/{{ openio_filebeat_namespace }}/filebeat-{{ openio_filebeat_serviceid }}"
+openio_filebeat_namespace: OPENIO
```
Comment (cdelgehier) [inline defaults/main.yml]: "{{ namespace | default('OPENIO') }}"

File: defaults/main.yml [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
+openio_filebeat_gridinit_dir: "/etc/gridinit.d"
+openio_filebeat_gridinit_file_prefix: "{{ openio_filebeat_namespace }}-"
```
Comment (cdelgehier) [inline defaults/main.yml]: openio_filebeat_gridinit_dir: "/etc/gridinit.d/{{ openio_filebeat_namespace }}"
openio_filebeat_gridinit_file_prefix: ""

File: defaults/main.yml [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+openio_filebeat_pid_dir: "/run/oio/sds/{{ openio_filebeat_namespace }}/{{ openio_filebeat_servicename }}.pid"
```
Comment (cdelgehier) [inline defaults/main.yml]: "/run/oio/sds/{{ openio_filebeat_namespace }}"

Comment (cdelgehier) [reply defaults/main.yml]: it's a var, not a default

File: defaults/main.yml [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+openio_filebeat_sysconfig_dir: "/etc/oio/sds/{{ openio_filebeat_namespace }}/{{ openio_filebeat_servicename }}"
```
Comment (cdelgehier) [inline defaults/main.yml]: it's a var

File: defaults/main.yml [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
 openio_filebeat_openio_log_path: "/var/log/oio/sds/{{ openio_filebeat_namespace }}"
+openio_filebeat_log_dir: "{{ openio_filebeat_openio_log_path }}/{{ openio_filebeat_servicename }}"
```
Comment (cdelgehier) [inline defaults/main.yml]: it's a var

File: defaults/main.yml [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
+openio_filebeat_volume: "/var/lib/oio/sds/{{ openio_filebeat_namespace }}/{{ openio_filebeat_servicename }}"
 
-openio_filebeat_es_hosts:
+openio_filebeat_elasticsearch_hosts:
```
Comment (cdelgehier) [inline defaults/main.yml]: localhost ?

Comment (cdelgehier) [reply defaults/main.yml]: Shouldn't you be using a group here?

File: defaults/main.yml (hunk 4 above)
Comment (cdelgehier) [reply defaults/main.yml]: and the value must be `openio_filebeat_sysconfig_dir: "/etc/oio/sds/{{ openio_filebeat_namespace }}"`

File: vars/main.yml [hunk 7]
```diff
@@ -1,27 +1,28 @@
...
 openio_filebeat_servicename: "filebeat-{{ openio_filebeat_serviceid}}"
-openio_filebeat_log_dir:
-  "/var/log/oio/sds/{{ openio_filebeat_namespace }}/{{ openio_filebeat_servicename }}"
-syslog_user: syslog
 
 openio_filebeat_default_inputs:
   - service: oioswift
     path: "{{ openio_filebeat_openio_log_path }}/oioswift-*/oioswift-*.log"
-    format: "%{} %{oio_host} %{} %{oio_loglevel}  %{oio_remote_addr} %{oio_local_addr} %{} %{oio_method} %{oio_url} %{} %{oio_status} %{} %{oio_user_agent} %{} %{oio_bytes_received} %{oio_bytes_sent} %{} %{oio_txid} %{} %{oio_resp_time_ms} %{oio_swift_source} %{} %{} %{} %{}"
+    format: "%{} %{oio_host} OIO,%{oio_namespace},oioswift,%{oio_service_id}: %{oio_loglevel}  %{oio_client_ip} %{oio_remote_addr} %{} %{oio_method} %{oio_url} %{} %{oio_status} %{} %{oio_user_agent} %{} %{oio_req_size} %{oio_resp_size} %{} %{oio_txid} %{} %{oio_resp_time} %{oio_swift_source} %{} %{oio_req_start_time} %{oio_req_end_time} %{oio_policy_index}"
```
Comment (cdelgehier) [inline vars/main.yml]: 120 chars max
```
format: "%{} %{oio_host} OIO,%{oio_namespace},\
         oioswift,%{oio_service_id}: %{oio_loglevel}  %{oio_client_ip} \
        ..."
```

**Review by cdelgehier — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 2 above)
Comment (fatpat) [reply defaults/main.yml]: why the (hell) all the roles are creating the directory `/etc/gridinit.d/{{ namespace }}` and the gridinit service files are then stored in the directory `/etc/gridinit.d` and named `{{ namespace }}-{{ service_name }}-{{ service_id }}.conf` ?

this is non sens

Comment (fatpat) [reply defaults/main.yml]: fixed

File: defaults/main.yml (hunk 3 above)
Comment (fatpat) [reply defaults/main.yml]: fixed

File: defaults/main.yml (hunk 4 above)
Comment (fatpat) [reply defaults/main.yml]: fixed

File: defaults/main.yml (hunk 5 above)
Comment (fatpat) [reply defaults/main.yml]: fixed

File: defaults/main.yml (hunk 6 above)
Comment (fatpat) [reply defaults/main.yml]: fixed

File: vars/main.yml (hunk 7 above)
Comment (fatpat) [reply vars/main.yml]: fixed

**Pull request merged**